chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SurveillanceReportController.java

```diff
@@ -257,8 +257,8 @@ public QuarterlyReport createQuarterlyReport(
         quarterlyReport.setQuarter(quarter);
         quarterlyReport.setActivitiesOutcomesSummary(createRequest.getSurveillanceActivitiesAndOutcomes());
         quarterlyReport.setPrioritizedElementSummary(createRequest.getPrioritizedElementSummary());
-        quarterlyReport.setReactiveSummary(createRequest.getReactiveSummary());
```

Comment:
Is there any possibility of either 1) Removing the DTO objects involved here, or 2) Refactoring to use Builder? Certainly understand if #1 is too big.